README.md

```diff
@@ -214,6 +215,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [osem](https://github.com/openSUSE/osem/labels/good%20first%20issue) _(label: good first issue)_ <br>Open Source Event Manager. An event management tool tailored to Free and Open Source Software conferences</br> See ["How to contribute to osem"](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md)
 - [open-build-service](https://github.com/openSUSE/open-build-service/labels/good%20first%20issue%20%3A1st_place_medal%3A) _(label: good first issue)_ <br>A generic system to build and distribute packages from sources in an automatic, consistent and reproducible way.</br> See ["How to contribute to open build service"](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md)
 - [bolt](https://github.com/puppetlabs/bolt/labels/Beginner%20Friendly) _(label: Beginner Friendly)_ <br>Bolt is a Ruby command-line tool for executing commands, scripts, and tasks on remote systems using SSH and WinRM.
+- [Jekyll](https://github.com/jekyll/jekyll/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp-wanted) _(label: help wanted)_ <br>Jekyll is a simple, blog-aware, static site generator perfect for personal, project, or organization sites in Ruby.
```

Comment:
This has a wrong URL format.

Also, this repo has both "good first issue" and "first-time-only", so it is not clear why "help wanted" should be added here. A clarification from repo maintainer is required.